.github/workflows/daily.yml

```diff
@@ -25,24 +25,41 @@ jobs:
         with:
           fetch-depth: 0         # Fetch all history for .GitInfo and .Lastmod
 
+      - name: Parse .tool-versions for hugo-version
+        id: hugo-version
+        run: |
+          grep 'hugo extended_' .tool-versions |
```

Comment:
I'm not overly familiar with asdf, but it looks like they publish [github actions](https://github.com/asdf-vm/actions) to install tools which not only saves us from grepping the file but might also help simplify the rest of the file. Is that useful?

Reply:
That's amazing! I'll check them out and update PR if it can be used